File: Chapter 6/Exercise/3. Integer.c
/*
 *  Name Program    : Integer
 *  Purpose         : Answer question from exercise Chapter 6 from Logic Algorithm and Basic Programming Book
 *  Author          : Yogi Siswanto
 *  Email           : yogisiswanto.c2@gmail.com
 *  Date Created    : 15th April 2021
 *  Language        : C Language
 */

#include <stdio.h>

// Main function
int main(){

    // Variable declaration
    int input[5];

    printf("Masukan lima buah angka secara berurutan, lalu tekan enter setelahnya\n");

    // Get input from console
    /* 
        Example
        1
        2
        3
        4
        5
     */
    scanf("%d", &input[0]);
    scanf("%d", &input[1]);
    scanf("%d", &input[2]);
    scanf("%d", &input[3]);
    scanf("%d", &input[4]);

    // Print to console
    printf("Angka ke - 1 = %d\n", input[0]);
    printf("Angka ke - 2 = %d\n", input[1]);
    printf("Angka ke - 3 = %d\n", input[2]);
    printf("Angka ke - 4 = %d\n", input[3]);
    printf("Angka ke - 5 = %d\n", input[4]);

    return 0;
}

File: Chapter 6/Exercise/3. Integer.java
/*
 *  Name Program    : Integer
 *  Purpose         : Answer question from exercise Chapter 6 from Logic Algorithm and Basic Programming Book
 *  Author          : Yogi Siswanto
 *  Email           : yogisiswanto.c2@gmail.com
 *  Date Created    : 15th April 2021
 *  Language        : Java Language
 */

import java.util.Scanner;

public class Main{

    // Main function
    public static void main(String[] args){

        // Variable instantiation
        int[] input = new int[5];
        Scanner sc = new Scanner(System.in);

        System.out.println("Masukan lima buah angka satu persatu, tekan enter setiap selesai memasukkan");

        // Get input from console
        /* 
            Example
            1
            2
            3
            4
            5
        */
        try{
            
            input[0] = sc.nextInt();
            input[1] = sc.nextInt();
            input[2] = sc.nextInt();
            input[3] = sc.nextInt();
            input[4] = sc.nextInt();
        
        }catch(Exception e){

        }

        // Print to console
        System.out.println("Angka ke 1 = " + input[0]);
        System.out.println("Angka ke 2 = " + input[1]);
        System.out.println("Angka ke 3 = " + input[2]);
        System.out.println("Angka ke 4 = " + input[3]);
        System.out.println("Angka ke 5 = " + input[4]);
    }
}